metal/resource_metal_connection.go

```diff
@@ -0,0 +1,172 @@
+package metal
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/packethost/packngo"
+)
+
+func resourceMetalConnection() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceMetalConnectionRead,
+		Create: resourceMetalConnectionCreate,
+		Delete: resourceMetalConnectionDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the connection resource",
+				// remove ForceNew and do Update
+				ForceNew: true,
+			},
+			"facility": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   "Facility where to create the connection",
```

Comment:
Same comment for metro
```suggestion
				Description:   "Facility where the connection will be created",
```